python/tank/bootstrap/manager.py

```diff
@@ -583,11 +585,41 @@ def get_pipeline_configurations(self, project):
             current_login=self._sg_user.login,
             sg_connection=self._sg_connection,
         ):
+            try:
+                path = ShotgunPath.from_shotgun_dict(pc)
+                config_path = path.current_os
+            except TankError:
+                log.warning(
+                    "Unable to determine config path for %s: %s",
```

Comment:
I wonder if instead of filtering bad pipeline configurations here we shouldn't do it in the resolver. We already filter out redundant Primary configurations and give a warning since it's a sign on an incorrect setup, maybe we should do the same with the pipeline configurations that don't have a path set for the current os.

What do you and @manneohrstrom think?